app/code/Magento/Customer/Controller/Ajax/Login.php

```diff
@@ -57,6 +57,11 @@ class Login extends \Magento\Framework\App\Action\Action
      * @var ScopeConfigInterface
      */
     protected $scopeConfig;
+    
+    /**
+     * @var url
+     */
+    protected $_url;
```

Comment:
1. New properties should be private
2. properties should not be prefixed with underscore